packages/happo-target-firefox/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { constructUrl } = require('happo-core');
+const runVisualDiffs = require('./runVisualDiffs');
+const initializeWebdriver = require('./initializeWebdriver');
+const closeDriver = require('./closeDriver');
+const checkBrowserVersion = require('./checkBrowserVersion');
+const server = require('./server');
```

Comment:
Maybe the server should be in its own package, or part of core? It's needed for the `review` and `upload` actions, which aren't specific to firefox. Or perhaps we just have two different servers, one for review and one used internally by the firefox target. That's probably a better separation of concerns than trying to cram everything into a single module. 

EDIT: It looks like you already did that. Nice!